mlrun/api/api/endpoints/functions.py

```diff
@@ -175,6 +176,7 @@ async def list_functions(
         project,
         name,
         tag,
+        hash_key,
```

Comment:
U seem to have a bug here - the crud list function has the `hash_key` as the last argument, while u pass it here the labels as the last, to avoid those scenarios I would suggest at first to get the crud function aligned with all others (except the endpoints) as last and define the arguments explicitly when calling the function

Reply:
I am quite surprised your tests passed with that bug, lets add a test case that covers that